File: practice/beginner/WATSCORE.cpp
/*
  Problem : That Is My Score!
  Author @ Rakesh Kumar
  cpp.rakesh@gmail.com
  Date : 29/08/2020
 */

#include <bits/stdc++.h>

int main() {
    std::ios_base::sync_with_stdio(false);
    std::cin.tie(0); std::cout.tie(0);

    int t = 0;
    std::cin >> t;
    while (t--) {
        int n = 0;
        std::cin >> n;
        std::vector<int> ank(13, 0);
        while (n--) {
            int p = 0, s = 0;
            std::cin >> p >> s;
            ank[p] = std::max(ank[p], s);
        }
        int total = 0;
        for (int i = 0; i <= 8; ++i)
            total += ank[i];
        std::cout << total << std::endl;
    }

    return 0;
}

File: practice/beginner/WATSCORE.py
'''
  Problem : That Is My Score!
  Author @ Rakesh Kumar
  cpp.rakesh@gmail.com
  Date : 26/02/2021
'''

def solve():
    for _ in range(int(input())):
        n = int(input())
        ank_talika = [0] * 8
        for _ in range(n):
            p, s = map(int, input().split())
            if p < 9:
                ank_talika[p - 1] = max(ank_talika[p - 1], s)
        print(sum(ank_talika))

if __name__ == '__main__':
    solve()
